crates/bevy_ecs/README.md

```diff
@@ -96,7 +96,7 @@ The built in "parallel stage" considers dependencies between systems and (by def
 Bevy ECS should feel very natural for those familiar with Rust syntax:
 
 ```rust
-use bevy_ecs::prelude::*;
+use bevy::prelude::*;
```

Comment:
Why do you want to make this change? This is an example for the `bevy_ecs` crate.

Reply:
That was what worked inside my IDE but when I looked at the changed lines I realised that it might be the standard for crate examples So I reverted it but you guys are really on the ball with code review :)